### PR TITLE
Enable SpecialGlobalVars cop

### PR DESCRIPTION
**Pull request by will-in-wi**
For discussion.

Followup to https://github.com/capistrano/capistrano/pull/1594

### CODE REVIEW

**Comment by mattbrictson** [comment]
I like this, but we have to be very careful to `require "English"` or it doesn't work. Perhaps RSpec or another dependency is doing it already, which is why tests pass. But without the `require` it may break for actual `cap` use.

File: capistrano.gemspec [hunk 1]
```diff
@@ -12,7 +12,7 @@ Gem::Specification.new do |gem|
   gem.summary       = "Capistrano - Welcome to easy deployment with Ruby over SSH"
   gem.homepage      = "http://capistranorb.com/"
 
-  gem.files         = `git ls-files`.split($/)
+  gem.files         = `git ls-files`.split($INPUT_RECORD_SEPARATOR)
```
Comment (mattbrictson) [inline capistrano.gemspec]: This might be safer, and doesn't require `English`: `git ls-files -z`.split("\x0")

**Comment by will-in-wi** [comment]
I added the English module to affected files.

**Comment by will-in-wi** [comment]
I also modified the gemspec per comment.

**Pull request merged**